CMakeLists.txt

```diff
@@ -98,10 +98,22 @@ if(OTIO_SHARED_LIBS)
 else()
     message(STATUS "Building static libs")
     set(OTIO_SHARED_OR_STATIC_LIB "STATIC")
-    if (OTIO_PYTHON_INSTALL AND NOT MSVC)
-        # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
-        set(CMAKE_CXX_VISIBILITY_PRESET hidden) 
-        set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+    if (OTIO_PYTHON_INSTALL)
+        if(MSVC AND Python_VERSION_MAJOR VERSION_LESS 3)
+            # Statically link run-time library (vcruntime and msvcp)
+            # See https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160
+            # This allows us to compile OTIO bindings with a newer MSVC version
+            # than the one used by the interpreter where OTIO will be installed.
+            # This is only required for Python < 3 because only these are
+            # compiled with an older compiler (9.0). CPython 3.5+ uses at least
+            # Visual C++ 14.X.
+            # See https://wiki.python.org/moin/WindowsCompilers#Which_Microsoft_Visual_C.2B-.2B-_compiler_to_use_with_a_specific_Python_version_.3F
+            add_compile_options(/MT)
+        else()
+            # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
```

Comment:
I'm not too sure why this `if(NOT MSVC)` statement was there before. Does someone remember why this was only done for non MSVC builds? If not I can test myself and see if we can apply this to all platforms.

Reply:
It's worth testing, the simpler this script gets the happier I am.

Reply:
From what I read in https://pybind11.readthedocs.io/en/stable/faq.html#how-can-i-create-smaller-binaries, symbols are hidden by default in Windows. And from what I see in the CMake code, it seems to be a no-op for MSVC compilers. So we could technically remove the condition.

Reply:
I went ahead and removed the condition.